### ISSUE TITLE
doc-note-box border

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-website#115]
gold #EED009 on #FAFAFC is 1.5:1

I think every place where we use that, it's a box border that is not required for understanding, and therefore doesn't need 3:1 ratio to meet WCAG.

However, it's good to have at least 4.5:1 ratio for better accessibility & usability for people with low contrast sensitivity *and* so people don't pick on our site for not meeting WCAG because they don't see that it's not required for understanding. (yes, 4.5:1 even though the non-text contrast SC is 3:1 because LVTF would prefer 4.5:1 and lets be exemplary)

Therefore, what about changing the border to the faded red in the Design Styles?

faded red #C0272D on #FAFAFC is 5.6:1

Also, I think the red is appropriate for most of those boxes.

**Comment by yatil** [comment]
I would not use a warning/error color for information purposes. We could use blue, but tbh I would not want those boxes more prominent. An alternate idea would be to just remove the box and use an icon and some other delimiter, then we don’t need to meet contrast requirements.

**Comment by slhenry** [comment]
I thought in the initial visual design that was a generic flourish color, not a warning color...

Although I agree it could be considered a warning/error color. -- which would be *good* for draft info box, but not for translations.

I *like* the box and would not want yet another icon.

Let's talk about options on Tuesday...

**Comment by yatil** [comment]
It depends on the usage of the color:

* Just somewhere in a graphic or as a bullet color -> likely flourish
* In a big box at the top of the document -> WARNING, WARNING

:-)

**Comment by slhenry** [comment]
OK, let's just leave it.

Let's do **document in the color table** that the gold on the light grays isn't sufficient contrast for it to be seen by some and thus "needed for understanding" -- to help us not forget and use it inappropriately in the future.

I started to do that myself... but couldn't re-find the color table. _cough_ documentation _cough_

**Comment by slhenry** [comment]
rtf, /me deleted the gold (for people who cannot see it) and the content works fine without it.
